pages/components/lists/lists.css

```diff
@@ -0,0 +1,50 @@
+.sui_ul {
+  list-style-type: none;
+  background: var(--WHITE);
+  border-radius: 4px;
+  width: fit-content;
+  padding: 1rem 2rem;
+  box-shadow: 0 0 2px 0 var(--BLACK-BG);
+}
+.sui_ul_title {
+  position: relative;
+  width: 100%;
+  padding: 1rem 0 2rem 0;
+}
+.sui_ul_title::after {
+  content: "";
+  width: 120%;
+  height: 1px;
+  position: absolute;
+  bottom: 1rem;
+  left: -10%;
+  background: var(--BLACK-BG);
+}
+.sui_ul li {
+  padding: 5px 0;
+  cursor: pointer;
+}
+.sui_li_p:hover {
+  color: var(--PRIMARY);
+}
+.sui_li_v1:hover {
+  color: var(--VARIANT-1-TXT-BG);
+}
+.sui_li_v2:hover {
+  color: var(--VARIANT-2-TXT-BG);
+}
+.sui_li_v3:hover {
+  color: var(--VARIANT-3-TXT-BG);
+}
+.sui_ul_stacked {
+  background: var(--SHADE2);
+  padding: 0;
+  width: 100%;
+}
+li.sui_li_stacked {
```

Comment:
Won't using the `sui_li_stacked` class work directly?

Reply:
It wont as the initial class ".sui_ul li" beats this one on specificity.

Reply:
Got it.